apps/files/src/utils/fileUtils.ts

```diff
@@ -21,25 +21,31 @@ export const extractFilePaths = function(path) {
 /**
  * Generate a translated summary of an array of nodes
  * @param {Node[]} nodes the nodes to summarize
+ * @param {number} hidden the number of hidden nodes
  * @return {string}
  */
-export const getSummaryFor = (nodes: Node[]): string => {
+export const getSummaryFor = (nodes: Node[], hidden = 0): string => {
 	const fileCount = nodes.filter(node => node.type === FileType.File).length
 	const folderCount = nodes.filter(node => node.type === FileType.Folder).length
 
+	let summary = ''
+
 	if (fileCount === 0) {
-		return n('files', '{folderCount} folder', '{folderCount} folders', folderCount, { folderCount })
+		summary = n('files', '{folderCount} folder', '{folderCount} folders', folderCount, { folderCount })
 	} else if (folderCount === 0) {
-		return n('files', '{fileCount} file', '{fileCount} files', fileCount, { fileCount })
-	}
-
-	if (fileCount === 1) {
-		return n('files', '1 file and {folderCount} folder', '1 file and {folderCount} folders', folderCount, { folderCount })
+		summary = n('files', '{fileCount} file', '{fileCount} files', fileCount, { fileCount })
+	} else if (fileCount === 1) {
+		summary = n('files', '1 file and {folderCount} folder', '1 file and {folderCount} folders', folderCount, { folderCount })
+	} else if (folderCount === 1) {
+		summary = n('files', '{fileCount} file and 1 folder', '{fileCount} files and 1 folder', fileCount, { fileCount })
+	} else {
+		summary = t('files', '{fileCount} files and {folderCount} folders', { fileCount, folderCount })
 	}
 
-	if (folderCount === 1) {
-		return n('files', '{fileCount} file and 1 folder', '{fileCount} files and 1 folder', fileCount, { fileCount })
+	if (hidden > 0) {
+		// TRANSLATORS: This is a summary of files and folders, where {hiddenFilesAndFolders} is the number of hidden files and folders
+		summary += n('files', ' ({hiddenFilesAndFolders} hidden)', ' ({hiddenFilesAndFolders} hidden)', hidden, { hiddenFilesAndFolders: hidden })
```

Comment:
Used a string concat anyway as the context is different. Covering all possible cases with two different plural variables is far too complex and not worth it

Reply:
Its not needed when you have multiple sentences. (you can always concat multiple sentences in languages, this is also correct for RTL and is even recommended for longer text blocks).

Reply:
Can you suggest an alternative then? Because I scratched my brain and couldn't figure one out :thinking:

Reply:
but do not have any leading whitespace.
```suggestion
		summary += ' ' + n('files', '(%n hidden)', ' (%n hidden)', hidden)
```

Reply:
@skjnldsv no I meant its correct what you are doing :)

Reply:
Ah, nice!
I'm reassured :sweat_smile:

Reply:
E.g. this would be correct:

```ts
const files = n('%n file', '%n files', numberFiles)
const folders = n('%n folder', '%n folders', numberFolders)
const hidden = n('%n hidden', '%n hidden', numberHidden)
let summary = ''
if (numberFiles > 0) {
    summary = files
}
if (numberFolders > 0) {
    if (summary !== '') {
        summary += ' • '
    }
    summary += folders
}
if (numberHidden > 0) {
    if (summary !== '') {
        summary += ' • '
    }
    summary += hidden
}
```

Which is equal to vue:
```html
<div>
    <span v-if="numberFiles">{{ n('%n file', '%n files', numberFiles) }}</span>
    <span v-if="numberFolders">
        <span v-if="numberFiles">•</span>
        {{ n('%n folder', '%n folders', numberFolders) }}</span>
    </span>
    <span v-if="numberHidden">
        <span v-if="numberFiles || numberFolders">•</span>
        {{ n('%n hidden', '%n hidden', numberHidden) }}</span>
    </span>
</div>
```